components/event-sources/cmd/http-adapter/main.go

```diff
@@ -97,27 +99,43 @@ func setupAdapter(component string, ector adapter.EnvConfigConstructor, ctor ada
 		logger.Error("error building statsreporter", zap.Error(err))
 	}
 
-	tracingCfg := DisabledTracingConfig
+	options := []cloudeventshttp.Option{
+		cloudevents.WithBinaryEncoding(),
```

Comment:
does this mean that the cloud event receiver will always except Binary Encoded request and we restrict the structured encoding?

Reply:
There are tests for both encodings:
- [structured](https://github.com/kyma-project/kyma/blob/master/components/event-sources/adapter/http/adapter_integration_test.go#L79)
- [binary](https://github.com/kyma-project/kyma/blob/master/components/event-sources/adapter/http/adapter_integration_test.go#L67)